### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by stefansc1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,3 +30,35 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - Supply
 - Basic tests
 - Documentation stub
+
+## [0.3.0] - TBD
+
+### Changed
+- general
+    - license changed to dual MIT/Apache-2.0
+    - use development branch of oemof-solph
```
Comment (j-brendel) [inline CHANGELOG.md:39]: - include oemof.thermal

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,3 +30,35 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+    - documentation moved to [Read the Docs](https://smooth.readthedocs.io)
+- components
+    - renamed Chp to H2Chp
+- functions
+    - optimization changed to multi-objective evolutionary algortihm based on NSGA-II, optionally with gradient ascent
+- models
+    - allow None value for fixed cost
+    - interval time is adjustable
```
Comment (j-brendel) [inline CHANGELOG.md:47]: - components can have 'variable' costs (CAPEX and OPEX), depending on the value of one of its parameters

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,3 +30,35 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
+### Added
+- general
+    - tests and examples for all components
```
Comment (j-brendel) [inline CHANGELOG.md:51]:
```suggestion
    - tests for all components
```
Unfortunately we don't have examples for all components yet

File: CHANGELOG.md [hunk 4]
```diff
@@ -30,3 +30,35 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+    - emissions
+- components
+    - BiogasConverter
+    - BiogasSmrPsa
+    - ElectrolyzerWasteHeat
+    - Gate
+    - PemElectrolyzer
+    - PowerConverter
```
Comment (j-brendel) [inline CHANGELOG.md:59]:
```suggestion
    - PowerConverter (replaces AC DC converter and DC AC inverter)
```

Comment (stefansc1) [reply CHANGELOG.md:62]: Neither were in v0.2.0

File: CHANGELOG.md [hunk 5]
```diff
@@ -30,3 +30,38 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - Supply
 - Basic tests
 - Documentation stub
+
+## [0.3.0] - 2021-04-08
+
+### Changed
+- general
+    - license changed to dual MIT/Apache-2.0
+    - use development branch of oemof-solph
```
Comment (j-brendel) [inline CHANGELOG.md:39]:
```suggestion
    - use development branch of oemof.solph
```

**Review by j-brendel — APPROVED** [review]

**Pull request merged**